Readme.md

```diff
@@ -156,8 +156,8 @@ File [./config/default.json5](config/default.json5) contains all configuration p
   register: {
     stake: "0.01", // amount of stake to set up
     funds: "0.02", // amount of funds to set up
-    mnemonic: "", // mnemonic to use for unlocking the account parameter
-    privateKey: "", // private key to retrieve the account address from
+    mnemonic: "", // mnemonic to use for unlocking the account parameter; DO NOT STORE IT HERE, use REGISTER_MNEMONIC as env variable. 
```

Comment:
I test it and works fine, I'm not sure if it's related with this PR, but if you send an invalid `privateKey` or `mnemonic`, it does not throw any error. 

I guess that we need to log the error in the catch.

Reply:
Yes, good catch. It wasn't related to this PR, but the error wasn't raised because the log level was set after retrieving the configuration. Now in the catch block, we log the error without the library to avoid this isue.